### PR TITLE
refactor(Enums): make property casing consistent

### PR DESCRIPTION
**Pull request by vaporox**
**Please describe the changes this PR makes and why it should be merged:**

This PR converts all enum properties to pascal case for more consistency within the library.

### CODE REVIEW

File: payloads/v8/channel.ts [hunk 1]
```diff
@@ -348,26 +348,26 @@ export interface APIMessage {
...
+	RecipientRemove,
+	Call,
+	ChannelNameChange,
+	ChannelIconChange,
+	ChannelPinnedMessage,
+	GuildMemberJoin,
+	UserPremiumGuildSubscription,
+	UserPremiumGuildSubscriptionTier1,
+	UserPremiumGuildSubscriptionTier2,
+	UserPremoimGuildSubscriptionTier3,
```
Comment (vladfrangu) [inline payloads/v8/channel.ts:362]: Bonk

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Review by kyranet — COMMENTED** [review]
Should `PermissionFlagsBits` be PascalCase as well? It's not an enum, but it behaves similar to one. cc: @vladfrangu 

https://github.com/vaporox/discord-api-types/blob/6dc030fe0cb5f44f15bdb62e738672f5ce50fbab/payloads/v8/permissions.ts#L15-L47

Other than that, there are no more SCREAMING_SNAKE_CASE keys besides v6's values, which are untouched by this PR.

**Review by iCrawl — APPROVED** [review]

**Comment by vladfrangu** [comment]
> Should `PermissionFlagsBits` be PascalCase as well? It's not an enum, but it behaves similar to one. cc: @vladfrangu
> 
> [vaporox/discord-api-types@`6dc030f`/payloads/v8/permissions.ts#L15-L47](https://github.com/vaporox/discord-api-types/blob/6dc030fe0cb5f44f15bdb62e738672f5ce50fbab/payloads/v8/permissions.ts#L15-L47)
> 
> Other than that, there are no more SCREAMING_SNAKE_CASE keys besides v6's values, which are untouched by this PR.

Yes! @vaporox ^

**Comment by vaporox** [comment]
Did that (with some unrelated things I found along the way as well), should the v6 types remain untouched?

File: deno/gateway/v8.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const GatewayVersion = '8';
 /**
  * https://discord.com/developers/docs/topics/opcodes-and-status-codes#gateway-gateway-opcodes
  */
-export enum GatewayOPCodes {
+export enum GatewayOpcodes {
```
Comment (kyranet) [inline deno/gateway/v8.ts:33]: *OPCode* stands for *OP Code*, which is an abbreviation of *Operation Code*, as such, the enum's name should be OpCode:
```suggestion
export enum GatewayOpCodes {
```

Comment (kyranet) [reply deno/gateway/v8.ts:33]: Also rename all references to it.

**Review by kyranet — CHANGES_REQUESTED** [review]

**Pull request merged**